docs/guides/how_to_guides/creating_batches/how_to_load_a_database_table_view_or_query_result_as_a_batch.rst

```diff
@@ -151,117 +150,140 @@ This guide shows how to get a :ref:`batch <reference__core_concepts__batches>` o
+                data_connectors:
+                  insert_your_runtime_data_connector_name_here:
+                    module_name: great_expectations.datasource.data_connector
+                    class_name: RuntimeDataConnector
+                    batch_identifiers:
+                      - some_key_maybe_pipeline_stage
+                      - some_other_key_maybe_run_id
 
-            .. code-block:: yaml
+        2. Obtain an Expectation Suite
 
-                insert_your_sqlalchemy_datasource_name_here:
-                  class_name: Datasource
-                  module_name: great_expectations.datasource
-                  execution_engine:
-                    class_name: SqlAlchemyExecutionEngine
-                    module_name: great_expectations.execution_engine
-                    connection_string: sqlite:///my_db_file # Insert your SqlAlchemy connection string here
-                  data_connectors:
-                    insert_your_runtime_data_connector_name_here:
-                      module_name: great_expectations.datasource.data_connector
-                      class_name: RuntimeDataConnector
-                      batch_identifiers:
-                        - some_key_maybe_pipeline_stage
-                        - some_other_key_maybe_run_id
+          .. code-block:: python
 
+            suite: ExpectationSuite = context.get_expectation_suite("insert_your_expectation_suite_name_here")
 
-        2. Obtain an Expectation Suite
+          Alternatively, you can simply use the name of the Expectation Suite.
 
-            .. code-block:: python
+          .. code-block:: python
 
-                suite = context.get_expectation_suite("insert_your_expectation_suite_name_here")
+            suite_name: str = "insert_your_expectation_suite_name_here"
 
-            Alternatively, if you have not already created a suite, you can do so now.
+          If you have not already created an Expectation Suite, you can do so now.
 
-            .. code-block:: python
+          .. code-block:: python
 
-                suite = context.create_expectation_suite("insert_your_expectation_suite_name_here")
+            suite: ExpectationSuite = context.create_expectation_suite("insert_your_expectation_suite_name_here")
 
-        3. Construct a BatchRequest
+        3. Construct a Runtime Batch Request
 
-            We will create a ``BatchRequest`` and pass it our DataFrame via the ``batch_data`` argument. The ``batch_identifiers`` argument is required and must be a non-empty dictionary containing all of the Batch Identifiers specified in your Runtime Data Connector configuration.
+          We will create a ``RuntimeBatchRequest`` and pass it our query via the ``runtime_parameters`` argument, under the ``query`` key. The ``batch_identifiers`` argument is required and must be a non-empty dictionary containing all of the Batch Identifiers specified in your Runtime Data Connector configuration.
```

Comment:
I think there might be instances in the DOC where RuntimeDataConnector and Runtime Data Connector / RuntimeBatchRequest and Runtime Batch Request are used interchangeably. One ask is that these be made consistent (which ever version it may be)